src/sentry/tasks/assemble.py

```diff
@@ -179,26 +190,43 @@ def get_assemble_status(task, scope, checksum):
     notice or error message.
     """
     cache_key = _get_cache_key(task, scope, checksum)
-    rv = default_cache.get(cache_key)
+
+    if options.get("assemble.read_from_redis"):
+        client = _get_redis_cluster_for_assemble()
+        rv = client.get(cache_key)
+
+        # It is stored as bytes with [state, detail] on Redis.
+        if rv:
+            [state, detail] = orjson.loads(rv)
+            rv = (state, detail)
```

Comment:
What is the difference between this and
```python
rv  = orjson.loads(rv)
```
?

Reply:
line 200 returns array, line 201 converts it into a tuple

Reply:
yes, you're right, actually we don't need this.